docs/api/causer-resolver.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Causer Resolver
+weight: 3
+---
+
+This class is registered as singleton and will allow you to set causer for activity globally or per action basis.
+
+**Note that** Overriding causer using `setCauser` method takes priority over overriding the resolver using `resolveUsing` method.
+
+```php
+CauserResolver::setCauser(User::find(1));
```

Comment:
See above comment

Reply:
@Gummibeer  suggested that we keep `Activity` class clean by separate defining activity causer in separate class.